input/assets/css/_cart.scss

```diff
@@ -322,6 +328,17 @@
 	.order-total {
 		font-size: 1.2em;
 	}
+
+  	.sm-float-right {
```

Comment:
@JulianSamarjiev The thing is, we want it to pull-right __only__ during a specific range of viewport-width (please note `.sm-`) :)

One thing I can agree on though, is have it named `.sm-pull-right`
Would that be good for you?